src/_locales/en_US/messages.json

```diff
@@ -23,6 +23,10 @@
         "message": "Currently Privacy Badger only checks if third parties are using cookies, HTML5 local storage, or canvas fingerprinting to track your browsing. Some of these domains may be using tracking methods that Privacy Badger can't detect.",
         "description": ""
     },
+    "options_enable_dnt_checkbox": {
+        "message": "Enable sending DNT header",
```

Comment:
Let's rename "DNT header" to "Do Not Track signal", and link it to https://www.eff.org/issues/do-not-track.

"Do Not Track" instead of "DNT" because we shouldn't assume familiarity with the abbreviation.

"Signal" instead of "header" because we also assert DNT in JavaScript (#1861).

Reply:
Done